cmake/Modules/ewoms-prereqs.cmake

```diff
@@ -36,10 +36,13 @@ set (ewoms_DEPS
 	"opm-parser"
 	"opm-grid"
 	"opm-core"
-	# librt (on some systems necessary for clock_gettime())
-	"librt REQUIRED"
 	# valgrind client requests
 	"Valgrind"
 	# quadruple precision floating point calculations
 	"Quadmath"
 	)
+
+if (NOT APPLE)
+	# librt (on some systems necessary for clock_gettime())
+	list(APPEND "librt REQUIRED" ewoms_DEPS)
+endif ()
```

Comment:
I think librt is not used anymore, because the timer code was changed to `std::chrono` for 2017.04.

Reply:
If so, I guess this could be removed entirely. Could you verify that?

Reply:
I just tested on my Ubuntu 16.04 machine: it works fine without librt.